pyproject.toml

```diff
@@ -11,6 +11,10 @@ classifiers = [
     "Operating System :: OS Independent",
 ]
 
+[project]
+name = "mallm"
+version = "v1.0.3"
```

Comment:
Why are you adding this? It's already defined above and can make problems with the auto deploy

Reply:
I got an error from ruff saying it is required for PEP 621 complicance